gpgwrapper/gpgwrapper.go

```diff
@@ -66,6 +74,21 @@ func (g *GnuPG) ImportArmoredKey(armoredKey string) (string, error) {
 	return output, nil
 }
 
```

Comment:
Ditto

Reply:
It's funny that, the *struct* isn't exported, but I kind of wanted to signify which methods you're actually supposed to use (only `PushLine` and `End`), but it doesn't really make sense exporting methods on an unexported object, I guess!

Reply:
I've unexported and documented `PushLine` and `End` now.